packages/hooks/src/useCfdAccounts.ts

```diff
@@ -0,0 +1,24 @@
+import useCFDAccounts from './useCFDAllAccounts';
+import useCFDDemoAccounts from './useCFDDemoAccounts';
+import useCFDRealAccounts from './useCFDRealAccounts';
+
+type TCfdAccountType = 'all' | 'real' | 'demo';
+
+/**
+ * this is a wrapper hook for useCFDDemoAccounts and useCFDRealAccounts
+ * and it returns the accounts list based on the given account_type
+ * you don't need to pass any argument if you need to get all CFD accounts
+ */
+
+const useGetCfdAccounts = (account_type: TCfdAccountType = 'all') => {
+    const all_cfd_accounts = useCFDAccounts();
+    const cfd_demo_accounts = useCFDDemoAccounts();
+    const cfd_real_accounts = useCFDRealAccounts();
+
+    if (account_type === 'demo') return cfd_demo_accounts;
+    else if (account_type === 'real') return cfd_real_accounts;
+
+    return all_cfd_accounts;
```

Comment:
I wanted to make it like this first but I thought it's more clear the way it is now. if u think this is more readable , I'll change it

Reply:
@shayan-deriv I'm suggesting this because we already calculate everything in the hook but then we throw it away based one the given type and return some part of it. So if we need both `real` and `demo` data we have to call the hook twice and calculate everything twice but only using some part of it, By returning an object we can call the hook once and also make the return type consistent 🤔

Reply:
agree. it's updated